src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -555,16 +556,37 @@ public function prependExtensionConfig($name, array $config)
      *  * Parameter values are resolved;
      *  * The parameter bag is frozen;
      *  * Extension loading is disabled.
+     *
+     * @param bool $resolveEnvPlaceholders Whether %env()% parameters should be resolved using the current
+     *                                     env vars or be replaced by uniquely identifiable placeholders
```

Comment:
That describes what it does, but I'd like to understand also *when* I should pass `true` here.